packages/gatsby/src/bin/cli.js

```diff
@@ -16,6 +17,18 @@ process.on(`unhandledRejection`, error => {
 
 const defaultHost = `localhost`
 
+let inGatsbySite = false
+let localPackageJSON
+try {
+  localPackageJSON = require(path.resolve(`./package.json`))
+  if (localPackageJSON.dependencies && localPackageJSON.dependencies.gatsby) {
```

Comment:
@KyleAMathews is there is a reason why `devDependencies` are not checked? in our setup `gatsby` is a dev dep, and this change broke our builds :crying_cat_face: .